sql/schema/v1-core.sql

```diff
@@ -571,20 +571,7 @@ BEGIN
         pcs.key
     FROM
         parent_to_child_strategy_ids pcs
-    ON CONFLICT (strategy_id, workflow_version_id, workflow_run_id) DO UPDATE
-        -- When concurrency slots are INSERTED:
-        -- We need to REMOVE their strategy_ids from the parent's completed_child_strategy_ids
-        -- because these child slots are now active again, not completed
-        -- This correctly handles bulk inserts by removing ALL strategy_ids in the current batch
-        SET completed_child_strategy_ids = ARRAY(
```

Comment:
We are no longer using `completed_child_strategy_ids` to case on whether to remove the workflow concurrency slot